gcp/modules/couchdb/values.yaml

```diff
@@ -21,3 +21,11 @@ affinity:
               values:
               - couchdb
         topologyKey: "kubernetes.io/hostname"
+
+resources:
+  requests:
+    cpu: 500m
+    memory: 256Mi
```

Comment:
I would strongly be in favor of setting the same values both for CPU & memory resources. This basically makes your cluster behave in a consistent manner (or more than this) and is kind of standard practice for this type of workloads.

Reply:
@stepanstipl I don't understand this. Can you please elaborate?

Reply:
Oh nevermind, for a moment I thought you're proposing to set CPU and memory requests to the same value 😆